test/pseudo-tty/no_dropped_stdio.js

```diff
@@ -12,4 +12,13 @@ let out = ('o'.repeat(48) + '\n').repeat(20);
 out += 'o'.repeat(24) + 'O';
 
 process.stdout.write(out);
-process.exit(0);
+
+// In AIX, the child exits even before the python parent
+// can setup the readloop. Provide a reasonable delay.
+if (common.isAix) {
+  setTimeout(function() {
+    process.exit(0);
+  }, 200);
```

Comment:
@addaleax - thanks, I understand the history of the test, and agree that the nextTick will defeat the purpose of the test. I have amended the code for no_dropped_stdio.js as per your suggestion. Looking at changes in other tests, I believe they are just fine.